functions/android.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         'subject', 'group', 'icon_url', 'ledColor', 'vibrationPattern', 'persistent', 
         'chronometer', 'when', 'alert_once', 'intent_class_name', 'notification_icon',
         'ble_advertise', 'ble_transmit', 'video', 'high_accuracy_update_interval',
-        'package_name'
+        'package_name', 'persistent'
```

Comment:
actually we need to get the command itself to be part of the rate limit exclusion a few lines below this :) `persistent` is already in this list

https://github.com/home-assistant/mobile-apps-fcm-push/blob/master/functions/android.js#L64-L67

Reply:
Oops! Thanks